index.js

```diff
@@ -34,12 +34,10 @@ module.exports = function(options) {
       headers: this.header,
       encoding: null,
       method: this.method,
-      body: parsedBody
+      body: parsedBody,
+      followRedirect: options.followRedirect || true
```

Comment:
followRedirect is `true` by default, the condition can be `options.followRedirect !== false`